exercises/practice/hello-world/.docs/instructions.append.md

```diff
@@ -1,6 +1,6 @@
 # Instructions append
 
-Since this is your first Java exercise, we've included a detailed TUTORIAL.md
+Since this is your first Java exercise, we've included a detailed [TUTORIAL.md](https://github.com/exercism/java/blob/main/exercises/practice/hello-world/TUTORIAL.md)
```

Comment:
So, we really do not want to be linking people over to the github pages. Instead, what we should do is move the contents from TUTORIAL.md into this instructions.append.md file. The whole idea of the TUTORIAL.md file I think goes back to older versions of exercism which relied more heavily on people completing the exercises locally at the command line and operated like a README. At least that is my interpretation of things. In any case, my recommendation stands.

Reply:
Thanks @jmrunkle, I will see the best option to complete that.